[2024-11-08T18:11:25.329+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-08T18:11:25.374+0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: telegram_text_dag.send_currency_rate scheduled__2024-11-08T00:00:00+00:00 [queued]>
[2024-11-08T18:11:25.408+0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: telegram_text_dag.send_currency_rate scheduled__2024-11-08T00:00:00+00:00 [queued]>
[2024-11-08T18:11:25.412+0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-08T18:11:25.454+0300] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): send_currency_rate> on 2024-11-08 00:00:00+00:00
[2024-11-08T18:11:25.476+0300] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-08T18:11:25.478+0300] {standard_task_runner.py:72} INFO - Started process 102 to run task
[2024-11-08T18:11:25.500+0300] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'telegram_text_dag', 'send_currency_rate', 'scheduled__2024-11-08T00:00:00+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/textMessageTG-dag.py', '--cfg-path', '/tmp/tmp5dnmoy4_']
[2024-11-08T18:11:25.507+0300] {standard_task_runner.py:105} INFO - Job 206: Subtask send_currency_rate
[2024-11-08T18:11:25.880+0300] {task_command.py:467} INFO - Running <TaskInstance: telegram_text_dag.send_currency_rate scheduled__2024-11-08T00:00:00+00:00 [running]> on host 1168f67a661e
[2024-11-08T18:11:26.472+0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='telegram_text_dag' AIRFLOW_CTX_TASK_ID='send_currency_rate' AIRFLOW_CTX_EXECUTION_DATE='2024-11-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-08T00:00:00+00:00'
[2024-11-08T18:11:26.482+0300] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-08T18:11:26.483+0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-08T18:11:26.484+0300] {logging_mixin.py:190} INFO - Current task name:send_currency_rate state:running start_date:2024-11-08 15:11:25.379401+00:00
[2024-11-08T18:11:26.485+0300] {logging_mixin.py:190} INFO - Dag name:telegram_text_dag and current dag run status:running
[2024-11-08T18:11:26.497+0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-08T18:11:26.923+0300] {python.py:240} INFO - Done. Returned value was: None
[2024-11-08T18:11:26.938+0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-08T18:11:26.940+0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=telegram_text_dag, task_id=send_currency_rate, run_id=scheduled__2024-11-08T00:00:00+00:00, execution_date=20241108T000000, start_date=20241108T151125, end_date=20241108T151126
[2024-11-08T18:11:26.941+0300] {taskinstance.py:1563} INFO - Executing callback at index 0: task_success_callback
[2024-11-08T18:11:26.943+0300] {baseoperator.py:405} WARNING - TelegramOperator.execute cannot be called outside TaskInstance!
[2024-11-08T18:11:26.954+0300] {base.py:84} INFO - Retrieving connection 'telegram_notifier'
[2024-11-08T18:11:27.475+0300] {_client.py:1773} INFO - HTTP Request: POST https://api.telegram.org/bot7997662746:AAF5-dAcYJmpsa1Pe5cqSENUoRV5D2lj5NA/sendMessage "HTTP/1.1 200 OK"
[2024-11-08T18:11:27.522+0300] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-08T18:11:27.524+0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-08T18:11:27.526+0300] {logging_mixin.py:190} INFO - Dag name:telegram_text_dag queued_at:2024-11-08 15:11:19.335543+00:00
[2024-11-08T18:11:27.527+0300] {logging_mixin.py:190} INFO - Task hostname:1168f67a661e operator:PythonOperator
[2024-11-08T18:11:27.587+0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-08T18:11:27.606+0300] {local_task_job_runner.py:245} INFO - ::endgroup::
